src/explore-education-statistics-common/src/modules/charts/components/__tests__/ChartRenderer.test.tsx

```diff
@@ -75,4 +79,42 @@ describe('ChartRenderer', () => {
     );
     expect(screen.queryByTestId('footnotes')).not.toBeInTheDocument();
   });
+
+  test('shows an error message with bad chart data ', () => {
+    const invalidTestChartRenderer: ChartRendererProps = {
+      type: 'verticalbar',
+      meta: testFullTableMeta.subjectMeta,
+      data: testFullTableMeta.results,
+      alt: '',
+      height: 100,
+      axes: undefined as unknown as LineChartProps['axes'],
```

Comment:
Minor, but for any of these type assertions, we could just simplify them to `undefined as never`. 

Same applies elsewhere in this PR.

Reply:
These tests are removed now.